src/components/Articles.jsx

```diff
@@ -1,13 +1,28 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+
 import ArticleList from './Lists/ArticleList';
+import { getTopics } from '../utils';
+import TopicSelect from './page-items/TopicSelect';
+import { useParams } from 'react-router-dom';
 
 export default function Articles() {
+	const [topics, setTopics] = useState([]);
```

Comment:
Since this useEffect simply hydrates the state for topics to be used as the options, might it not be better suited to living in the Topics component itself? This may lead to some logic changes, but just a thought